server/controllers/arrivalsController.test.ts

```diff
@@ -28,10 +29,41 @@ describe('ArrivalsController', () => {
         bookingId: 'bookingId',
         premisesId: 'premisesId',
       }
+      ;(fetchErrorsAndUserInput as jest.Mock).mockImplementation(() => {
```

Comment:
meh don't need to mockImplementation if just returning a value can just `mockReturn`